public-api/src/main/webapp/WEB-INF/appengine-web.xml.template

```diff
@@ -1,7 +1,8 @@
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
-  <service>public-api</service>
```

Comment:
I believe this change would cause workbench and db APIs to deploy to the same GAE service.

Reply:
Oops! Good catch. Fixed.